src/cmd/root.go

```diff
@@ -21,14 +22,11 @@ var v *viper.Viper
 var rootCmd = &cobra.Command{
 	Use: "zarf [COMMAND]",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		setLogLevel()
-		config.CliArch = arch
-
-		// Disable progress bars for CI envs
-		if os.Getenv("CI") == "true" {
-			message.Debug("CI environment detected, disabling progress bars")
-			message.NoProgress = true
+		// Don't add the logo to the help command
+		if cmd.Parent() == nil {
+			skipLogFile = true
 		}
+		alwaysRun()
```

Comment:
initialSetup() may be a better name for this function than alwaysRun() since that doesn't really explain what it is doing.